src/antares/service/local_services/renewable_local.py

```diff
@@ -29,12 +35,40 @@ def update_renewable_properties(
         self, renewable_cluster: RenewableCluster, properties: RenewableClusterProperties
     ) -> RenewableClusterProperties:
         raise NotImplementedError
+    
+    def _read_timeseries(self, ts_file_type: TimeSeriesFileType, study_path: Path, area_id: Optional[str] = None, renewable_id: Optional[str] = None) -> pd.DataFrame:
```

Comment:
You should put this method inside a utils as it's a duplicate from the one inside area_local